.gitignore

```diff
@@ -6,6 +6,9 @@
 *.dylib
 bin
 testbin/*
+api/*/zz*generated.deepcopy.go
```

Comment:
can we avoid removing this for now? I'm not sure what kind of magic will be required for eks-a to work without it. Currently, this is all we're running before expecting the repo to be functional: https://github.com/maxdrib/eks-anywhere-build-tooling/blob/main/projects/aws/cluster-api-provider-cloudstack/Makefile#L22-L24

Reply:
On line 169 `generate-mocks: bin/mockgen generate-deepcopy pkg/mocks/mock_client.go ` generate mocks generates deepcopy as a prerequisite. You shouldn't have to change the EKS-A command. 

Please let me know if you do.